### PR TITLE
Update Zookeeper version to 3.4.10 and add centos prep step

### PR DESCRIPTION
**Pull request by kaarolch**
* Update the zookeeper version to 3.4.10 
* Add centos prep steps to the makefile

### CODE REVIEW

**Review by tsauce — COMMENTED** [review]
👍 This matches the latest move for dc/os zookeeper version as well (as far back as 3 months ago)

**Review by tsauce — APPROVED** [review]

**Review by vipram — APPROVED** [review]

**Comment by jgehrcke** [comment]
Thanks for the patch. Superficially this looks very good and appropriate, of course, but I am not in the position to assess corresponding risks (if there are any).

@lingmann @lloesche @spahl -- opinions?

**Comment by wdhowe** [comment]
Any news on this?

Our vulnerability scanner is complaining about out dated zookeeper on our mesos masters. (says that the fixed version is 3.4.10)

**Comment by jgehrcke** [comment]
@artemharutyunyan @lloesche any input?

File: Makefile [hunk 1]
```diff
@@ -90,3 +90,8 @@ $(TOOR)/usr/lib/systemd/system/zookeeper.service: zookeeper.service
 prep-ubuntu:
 	sudo apt-get -y install ruby-dev rpm
 	sudo gem install fpm
+
+.PHONY: prep-centos
+prep-centos:
+	sudo dnf install ruby-devel gcc curl libyaml-devel rubygems rpm-build
```
Comment (lloesche) [inline Makefile:96]: Should this be `yum -y` on CentOS?

**Comment by kaarolch** [comment]
I can add it today afternoon.

wt., 28.08.2018, 18:05 użytkownik Lukas Lösche <notifications@github.com>
napisał:

**Comment by kaarolch** [comment]
I'm no longer work at zooplus so I create new PR with new stable version of zookeeper. #5

**Pull request closed without merging**